roles/DCOS.bootstrap/templates/docker/proxy.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Service]
+{% for key, value in docker_proxy_env.iteritems() %}
+Environment="{{ key }}={{ value }}"
```

Comment:
With this the naming is confusing. Its not only setting proxy environment variable but any. So either you should be explicit setting the proxy environment variables or `docker_proxy_env` should be called `dockerd_environment_variables`